tests/specs/infinite-gas.k

```diff
@@ -81,8 +74,12 @@ module INFINITE-GAS-JAVA [kast]
     rule (S1 -Int C2) -Int C3 => S1 -Int (C2 +Int C3) requires #isConcrete(C2) andBool #isConcrete(C3) andBool notBool #isConcrete(S1) [simplification]
 endmodule
 
-module INFINITE-GAS-COMMON
+module INFINITE-GAS-LEMMAS-COMMON
     imports EVM
+    imports INFINITE-GAS
+
+    syntax Int ::= #gas ( Int , Int , Int ) // startGas, nonMemory, memory
+ // -----------------------------------------------------------------------
```

Comment:
```suggestion
    syntax Int ::= #gas ( Int , Int , Int ) [function, no-evaluators] // startGas, nonMemory, memory
 // ------------------------------------------------------------------------------------------------
```

Reply:
Actually, nevermind, this isn't needed because of the `macro` rule the haskell backend sees.